spec/controllers/donations_controller_spec.rb

```diff
@@ -160,22 +160,6 @@
         expect(subject).to redirect_to(dashboard_path)
       end
     end
-
```

Comment:
Please explain why you are removing these tests.  On the surface it seems like we would still want to make sure that you can't access donations for a different org, and that you have to be signed in to access the donations

Reply:
True - let me rejig these instead. The issue is that it was depending on the organization\_id in the parameter, which is no longer the real indication of which organization you're looking at. In fact, we could remove that organization\_id which we add to every URL - it makes it a bit confusing.

Reply:
Ah I know why I removed them - because I added application-level tests instead. So rather than testing a single controller we're now testing every controller in the app :smile:

Reply:
Alright, then.